spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  let(:valid_attributes) do
+    {
+      name: 'Mark'
+    }
+  end
+
+  describe 'GET /index' do
+    before :each do
+      @user = User.create! valid_attributes
+      get user_posts_path(@user)
+    end
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
```

Comment:
- [ ] Good job for adding each action you check: If response status was correct, If a correct template was rendered and If the response body includes correct placeholder text But Kindly makes sure that all test cases are passing when I run on my terminal I got one of your test case fail So kindly Fix it.
![test](https://github.com/batoolfatima2135/Blog-app/assets/93598189/86e37e43-6254-492a-8a06-60af146ee55b)


![test 1](https://github.com/batoolfatima2135/Blog-app/assets/93598189/767f1e9b-5894-46f4-a616-53adecc35d7b)

Reply:
Hi @Abenezer-Tilahun , on my end every test is passing!
![Capture](https://github.com/batoolfatima2135/Blog-app/assets/61277612/5c8bcec3-9f7a-499d-99da-ebdf12c3e04d)
What should I do now?? 🤔